xml/admin_ceph_datamgm.xml

```diff
@@ -1451,30 +1451,88 @@ osdmap e13 pg 1.6c (1.6c) -> up [1,0] acting [1,0]
     backfill of the group is done.
    </para>
   </sect2>
-
   <sect2 xml:id="op-pgs-revert">
-   <title>Reverting Lost Objects</title>
-   <para>
-    If the cluster has lost one or more objects and you have decided to abandon
-    the search for the lost data, you need to mark the unfound objects as
-    'lost'.
-   </para>
-   <para>
-    If the objects are still lost after having queried all possible locations,
-    you may need to give up on the lost objects. This is possible given unusual
-    combinations of failures that allow the cluster to learn about writes that
-    were performed before the writes themselves are recovered.
-   </para>
-   <para>
-    Currently the only supported option is 'revert', which will either roll
-    back to a previous version of the object, or forget about it entirely in
-    case of a new object. To mark the 'unfound' objects as 'lost', run the
-    following:
-   </para>
-<screen>
-&prompt.root;ceph pg <replaceable>PG_ID</replaceable> mark_unfound_lost revert|delete
-</screen>
-  </sect2>
+     <title>Reverting Lost Objects</title>
+     <para>
+      If the cluster has lost one or more objects and you have decided to abandon
+      the search for the lost data, you need to mark the unfound objects as
+      'lost'.
+     </para>
+     <para>
+      If the objects are still lost after having queried all possible locations,
+      you may need to give up on the lost objects. This is possible given unusual
```

Comment:
This sentence is long and complicated, I don't understand it 100%

Reply:
Hmm actually, I think lines 1462 - 1465 are kinda redundant. I think we could remove them. Thoughts?

Reply:
ok now